source/agora/test/Base.d

```diff
@@ -442,6 +442,29 @@ public class TestAPIManager
         this.nodes = null;
     }
 
+    /***************************************************************************
+
+        Restart a specific node
+
+        This routine restarts the node at the given `index`, making sure it
+        gracefully shuts down then restart properly.
+
+        Params:
+          index = Index of the node to restart
+
+    ***************************************************************************/
+
+    public void restart (size_t index)
```

Comment:
Could this take a client as a parameter instead?

```D
auto node = network.clients[4];
network.restart(node);
```

I think using an index could be a little awkward to use?

Reply:
Doh. So obvious now you mention it.